moveit_ros/planning_interface/move_group_interface/include/moveit/move_group_interface/move_group_interface.h

```diff
@@ -740,30 +740,113 @@ class MoveGroupInterface
+                              double revolute_jump_threshold, double prismatic_jump_threshold,
+                              moveit_msgs::RobotTrajectory& trajectory,
+                              const moveit_msgs::Constraints& path_constraints, bool avoid_collisions = true,
+                              moveit_msgs::MoveItErrorCodes* error_code = nullptr)
+  {
+    return computeCartesianPath(waypoints, eef_step, 0, revolute_jump_threshold, prismatic_jump_threshold, trajectory,
+                                path_constraints, avoid_collisions, error_code);
+  }
+
+  /** \brief Compute a Cartesian path that follows specified waypoints with a step size of at most \e eef_step meters
+      between end effector configurations of consecutive points in the result \e trajectory. The reference frame for the
+      waypoints is that specified by setPoseReferenceFrame(). No more than \e revolute_jump_threshold radians
+      is allowed on each joint as change in distance in the configuration space of the robot.  Similarly, no more than
+      \e prismatic_jump_threshold m between waypoints is allowed. This is to prevent 'jumps' in IK solutions.
+      \e jump_threshold defines a factor off the average configuration space distance between waypoints that is
+      acceptable between any two waypoints.
+      Collisions are avoided if \e avoid_collisions is set to true. If collisions cannot be avoided, the function fails.
+      Return a value that is between 0.0 and 1.0 indicating the fraction of the path achieved as described by the
+      waypoints.
+      Return -1.0 in case of error. */
+  double computeCartesianPath(const std::vector<geometry_msgs::Pose>& waypoints, double eef_step, double jump_threshold,
+                              double revolute_jump_threshold, double prismatic_jump_threshold,
                               moveit_msgs::RobotTrajectory& trajectory, bool avoid_collisions = true,
                               moveit_msgs::MoveItErrorCodes* error_code = nullptr);
 
   /** \brief Compute a Cartesian path that follows specified waypoints with a step size of at most \e eef_step meters
       between end effector configurations of consecutive points in the result \e trajectory. The reference frame for the
-      waypoints is that specified by setPoseReferenceFrame(). No more than \e jump_threshold
-      is allowed as change in distance in the configuration space of the robot (this is to prevent 'jumps' in IK
-     solutions).
+      waypoints is that specified by setPoseReferenceFrame(). No more than \e revolute_jump_threshold radians
+      is allowed on each joint as change in distance in the configuration space of the robot.  Similarly, no more than
+      \e prismatic_jump_threshold m between waypoints is allowed. This is to prevent 'jumps' in IK solutions.
+      \e jump_threshold defines a factor off the average configuration space distance between waypoints that is
+      acceptable between any two waypoints.
```

Comment:
```suggestion
      waypoints is that specified by setPoseReferenceFrame(). \e revolute_jump_threshold [rad]
      and \e prismatic_jump_threshold [m] limit the absolute jump distance between waypoints for each joint. \e jump_threshold defines the maximum permissible averaged distance between waypoints in configuration space may not exceed. This is to prevent 'jumps' in IK solutions.
```
Is `jump_threshold` the maximum average distance in c-space, or a factor between two joint configurations? I found that difficult to parse. 

The block was overall unwieldy, so here's a suggestion. The old version works too, except for the typo "off".